opba-db/src/main/java/de/adorsys/opba/db/domain/entity/fintech/FintechRequest.java

```diff
@@ -0,0 +1,37 @@
+package de.adorsys.opba.db.domain.entity.fintech;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.SequenceGenerator;
+
+@Getter
```

Comment:
Please add FIXME and a ticket that it is temporary solution until we achieve all request fields signing

Reply:
The class was removed